### ISSUE TITLE
Yarn validate/Lint running out of memory

### ISSUE THREAD

**Issue by KolbyRKunz** [hyperledger/aries-framework-javascript#1234]
With the update to AFJ 0.3.x whenever I run the yarn validate script it crashes on the yarn lint command. 

`FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
 1: 0x1006b58d8 node::Abort() [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
 2: 0x1006b5a60 node::ModifyCodeGenerationFromStrings(v8::Local<v8::Context>, v8::Local<v8::Value>, bool) [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
 3: 0x1007f736c v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
 4: 0x1007f7300 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
 5: 0x10097a81c v8::internal::Heap::GarbageCollectionReasonToString(v8::internal::GarbageCollectionReason) [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
 6: 0x10097933c v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
 7: 0x100984684 v8::internal::Heap::AllocateRawWithLightRetrySlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
 8: 0x100984718 v8::internal::Heap::AllocateRawWithRetryOrFailSlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
 9: 0x1009575ac v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType, v8::internal::AllocationOrigin) [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
10: 0x100c8f6d0 v8::internal::Runtime_AllocateInYoungGeneration(int, unsigned long*, v8::internal::Isolate*) [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]
11: 0x100fa2b8c Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_NoBuiltinExit [/Users/kolbykunz/.nvm/versions/node/v16.17.0/bin/node]`

I have attempted to increase the amount of memory that the node heap has to work with but it continues to ask for more and more memory beyond 8GB's

**Comment by TimoGlastra** [comment]
Yeah not sure what's up with ESLint, but it takes longer to run than the actual tests. Maybe we can speed it up a bit and reduce memory!

**Comment by blu3beri** [comment]
Shot in the dark here, but maybe `eslint_d` will do it slightly better?

**Comment by KolbyRKunz** [comment]
I checked the version of ESlint between 0.3.x and 0.2.5 and they are the same version so unless a dependency has changed I am not sure what could have broken it.
